templates/web/bromley/reports/_problem-list.html

```diff
@@ -0,0 +1,19 @@
+<section class="full-width">
```

Comment:
This template is identical to the parent, and doesn't need to be added, I think.